test/functional/uuid/jsonb.ts

```diff
@@ -7,10 +7,17 @@ import {createTestingConnections, closeTestingConnections} from "../../utils/tes
 describe("uuid type", () => {
 
     let connections: Connection[];
-    before(async () => connections = await createTestingConnections({
-        entities: [Record],
-        enabledDrivers: ["postgres"] // because only postgres supports jsonb type
-    }));
+    before(async () => {
+        connections = await createTestingConnections({
+            entities: [Record],
+            enabledDrivers: ["postgres"] // because only postgres supports jsonb type
+        });
+
+        connections.forEach(async connection => {
+            let queryRunner = await connection.driver.createQueryRunner();
+            await queryRunner.query(`CREATE extension IF NOT EXISTS "uuid-ossp"`);
```

Comment:
when you are creating your own query runner you need to release it. Probably thats why your test is hanging. But, probably better here just to call `connection.entityManager.query()` instead